src/combinators/TWAP.sol

```diff
@@ -42,6 +43,10 @@ contract TWAP is Ward {
             first.tally = 0;
         }
         configs[tag] = _config;
+
+        //  if targettag is empty, just use same tag
+        if (_config.ttag == bytes32(0)) configs[tag].ttag = tag;
```

Comment:
don't think making it optional adds much value, better to make the user think about the sources of their data

Reply:
resolved